build/android/cross

```diff
@@ -125,10 +125,10 @@ export LDFLAGS="${LDFLAGS} -L${SYSROOT}/usr/lib${LIB_SUFFIX} -L${ANDROID_TOOLCHA
 export pkg_configure_flags="--host=${TOOLCHAIN_NAME} --disable-shared $CONFIG_EXTRA"
 
 # Where to install headers and libraries
-export pkg_prefix="${ROOTDIR}/jni"
+export pkg_prefix="${ROOTDIR}"
 
 # Tell ./build/dependency that we are cross compiling
-export pkg_cross="android"
+export pkg_cross="jni"
```

Comment:
Names here changed to make sure we output in `${ROOTDIR}/jni`. Semantic of `pkg_cross` changed from generically meaning "we're cross compiling for platform" to mean exactly the type of platform ("jni" for Android and one of "iphoneos" and "iphonesimulator" for iOS).